docs/upgrade_1-4_to_1-5.md

```diff
@@ -0,0 +1,46 @@
+# Upgrading from OADP 1.4
+
+> **NOTE:** Always upgrade to next minor version, do NOT skip versions. To update to higher version, please upgrade one channel at a time. Example: to upgrade from 1.1 to 1.3, upgrade first to 1.2, then to 1.3.
+
+## Changes from OADP 1.4 to 1.5
+
+- Velero was updated from version 1.14 to 1.16 (Changes reference: https://velero.io/docs/v1.15/upgrade-to-1.15/ https://velero.io/docs/v1.16/upgrade-to-1.16/)
```

Comment:
Shouldn't we hold off from adding upstream upgrade documentation ? (might confuse the users, did you mean to add the release notes/changelog summary here ?)

Reply:
this is to tell changes in velero, we have been following this approach in other upgrade docs 
    https://github.com/openshift/oadp-operator/blob/master/docs/upgrade_1-1_to_1-2.md
    https://github.com/openshift/oadp-operator/blob/master/docs/upgrade_1-2_to_1-3.md
    https://github.com/openshift/oadp-operator/blob/master/docs/upgrade_1-3_to_1-4.md

Reply:
right, understood your point

I think goal here is to help us remember what change we must tell user to do (in DPA, for example) when updating OADP

Adding release notes would be good, but that would be a lot of links, right? from 1.15.0 until 1.16.0

maybe future enhancement?

Reply:
alright, added both :smile: